### PR TITLE
GEOPY-1414: add property group suffix to las export files

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-1414 - add property group suffix to las export files**

### CODE REVIEW

File: las_geoh5/export_files/driver.py [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
+    """
+
+    if isinstance(basepath, str):
+        basepath = Path(basepath)
+
+    drillholes = [k for k in group.children if isinstance(k, Drillhole)]
+
+    # subpath = basepath / group.name
+    # if not subpath.exists():
+    #     subpath.mkdir()
```
Comment (domfournier) [inline las_geoh5/export_files/driver.py:50]: Remove commented code

File: las_geoh5/export_files/uijson.py
Comment (domfournier) [inline las_geoh5/export_files/uijson.py:1]: I personally would be tempted to only keep the file in assets - less to maintain.

File: las_geoh5/import_files/driver.py [hunk 2]
```diff
@@ -65,6 +89,9 @@ def run(filepath: str):  # pylint: disable=too-many-locals
     workspace = Workspace()
     begin_reading = time()
 
+    # lasfiles = []
+    # for file in ifile.data["files"].split(";"):
+    #     lasfiles.append(lasio.read(file, mnemonic_case="preserve"))
```
Comment (domfournier) [inline las_geoh5/import_files/driver.py:94]: Clean up

**Review by domfournier — COMMENTED** [review]
Looks good. Minor comments below

File: las_geoh5/export_files/driver.py (hunk 1 above)
Comment (benk-mira) [reply las_geoh5/export_files/driver.py:50]: whoops, thanks for catching!

File: las_geoh5/export_files/uijson.py
Comment (benk-mira) [reply las_geoh5/export_files/uijson.py:1]: yeah, sounds good

Comment (benk-mira) [reply las_geoh5/export_files/uijson.py:1]: Let's open a new issue.  Attempted but ran into roadblocks re-writing tests.

**Review by domfournier — APPROVED** [review]

**Review by sebhmg — APPROVED** [review]

**Pull request merged**